Author: Cesare Falco <c.falco@ubuntu.com>
Description: Set webserver document root.
 Set internal webserver document root to a more convenient value.
Forwarded: no

Index: mame/src/emu/emuopts.c
===================================================================
--- mame.orig/src/emu/emuopts.c	2013-09-17 08:47:03.000000000 +0200
+++ mame/src/emu/emuopts.c	2013-09-27 15:37:52.000000000 +0200
@@ -203,7 +203,7 @@
 	{ OPTION_AUTOBOOT_SCRIPT ";script",                  NULL,        OPTION_STRING,     "lua script to execute after machine boot" },
 	{ OPTION_HTTP,                                       "0",         OPTION_BOOLEAN,    "enable local http server" },
 	{ OPTION_HTTP_PORT,                                  "8080",      OPTION_INTEGER,    "http server listener port" },
-	{ OPTION_HTTP_PATH,                                  "web",       OPTION_STRING,     "path to web files" },
+	{ OPTION_HTTP_PATH,                                  "/usr/share/games/mess/web",       OPTION_STRING,     "path to web files" },
 	{ NULL }
 };
 
